cmake/dependencies/common.cmake

```diff
@@ -51,10 +51,10 @@ if(NOT DEFINED FFMPEG_PREPARED_BINARIES)
     endif()
     set(FFMPEG_LIBRARIES
             "${FFMPEG_PREPARED_BINARIES}/lib/libavcodec.a"
+            "${FFMPEG_PREPARED_BINARIES}/lib/libswscale.a"
```

Comment:
`libswscale.a` is now pulling in several functions from `libavutil.a`, so linking order needed to be changed to handle that.

Reply:
This probably needs to be changed in the else block just below as well.

Reply:
Yep, good catch.